model/rules.py

```diff
@@ -0,0 +1,45 @@
+from enum import Enum
+import abstract.guards as guards
+
+
+class RuleType(Enum):
+    """Collection of rule types used to uniquely identify each subclass of Rule. The Enum's value is what should show up in the rule YAML file"""
```

Comment:
Usage of the word "Collection" can lead to confusion.
By definition, this is not a collection.